### PR TITLE
Istep 13 8

### PR DESCRIPTION
**Pull request by krystian-hebel**

### CODE REVIEW

**Comment by macpijan** [comment]
@miczyg1 Please review

File: src/mainboard/raptor-cs/talos-2/romstage.c [hunk 1]
```diff
@@ -8,39 +8,333 @@
...
+					break;
+				case 0x5:
+					mem_data.mcs[mcs].mca[mca].nrfc_dlr = ns_to_nck(120);
+					break;
+				case 0x6:
+					mem_data.mcs[mcs].mca[mca].nrfc_dlr = ns_to_nck(185);
+					break;
+				default:
+					die("Unsupported DRAM density\n");
+			}
```
Comment (miczyg1) [inline src/mainboard/raptor-cs/talos-2/romstage.c:300]: The code from lines 209 to 300 could be a function. This kind of nested level is hard to follow when a for loop is so long

**Review by miczyg1 — CHANGES_REQUESTED** [review]

**Comment by macpijan** [comment]
Already integrated in: https://github.com/3mdeb/coreboot/pull/49
